app/front-end/src/features/editor/components/editorView/editorView.tsx

```diff
@@ -127,7 +127,6 @@ export const EditorView: React.FC = () => {
 
   const onCellEditStart = () => {
     unsavedStateUpdate(true);
-    console.log(unsaved);
   };
 
   const getWorkspaceFile = useCallback(async () => {
```

Comment:
Also when moving to the next or previous page of the file, it should also prompt that changes may be lost.
The same principle should also be done with changing page size number.

Reply:
Resolved